README.rst

```diff
@@ -74,6 +74,14 @@ be used to create new transcodes. It is assumed that your compiled
 version of ffmpeg has the matching codec libraries required for the
 transcode.
 
+Settings:
+~~~~~~~~~
+
+The following settings are available to control the behavior of the module.
+
+* `WAGTAILVIDEOS_MAX_UPLOAD_SIZE` - Maximum upload size of video files in bytes. Defaults to 1GB
```

Comment:
I can't see this setting used anywhere

Reply:
It actually already exists in the fields:

https://github.com/neon-jungle/wagtailvideos/blob/master/wagtailvideos/fields.py#L13

As a I was diving through the code I just happened to notice that it was not documented and added it to the README. I can pull this doc change out of the branch if you prefer, as it technically does not relate to the nature of the pull request.